### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

**Review by SevKrok — APPROVED** [review]
OK

File: db/models.py [hunk 1]
```diff
@@ -50,3 +59,75 @@ class MovieSession(models.Model):
...
+        to=AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="orders"
+    )
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+    class Meta:
+        ordering = ["-created_at"]
```
Comment (fsocie7y) [inline db/models.py:76]: class Meta should be placed before methods. fix it everywhere

File: services/user.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+
+
+def get_user(user_id: int) -> User:
```
Comment (fsocie7y) [inline services/user.py:27]: u can use `get_user_model` for type annotation too)

**Review by fsocie7y — CHANGES_REQUESTED** [review]

File: services/user.py (hunk 2 above)
Comment (Daniil-Pankieiev) [reply services/user.py]: Fixed

**Review by fsocie7y — APPROVED** [review]
GJ 👍

**Pull request closed without merging**